.github/workflows/pull-request-ci.yaml

```diff
@@ -0,0 +1,307 @@
+          experimental: true
+
+    steps:
+    - name: Checkout KubeCF
+      uses: actions/checkout@v2
+      with:
+        path: kubecf
+        submodules: recursive
+    - name: Checkout catapult
+      uses: actions/checkout@v2
+      with:
+        repository: SUSE/catapult
+        path: catapult
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: cache.tools
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.workspace }}/tools
+        key: ${{ runner.os }}-tools
+    - run: make tools-install
+
+    - id: download-bundle
+      name: Download chart bundle
+      uses: actions/download-artifact@v2
+      with:
+        name: kubecf-bundle.tgz
+        # The artifact upload/download creates an extra directory
+        path: ${{ github.workspace }}/kubecf-bundle
+
+    - name: Download gomplate
+      run: |
+        if test -x "${TOOLS_DIR}/gomplate" ; then
+          echo "gomplate already exists; skipping."
+          exit 0
+        fi
+        mkdir -p "${TOOLS_DIR}"
+        wget -O "${TOOLS_DIR}/gomplate" \
+          https://github.com/hairyhenderson/gomplate/releases/latest/download/gomplate_linux-amd64
+        chmod a+x "${TOOLS_DIR}/gomplate"
+
+    - name: Start SSH agent
+      run: |
+        set -o errexit -o pipefail -o nounset
+        eval "$(ssh-agent -s)"
+        ssh-keygen -t rsa -b 4096 -N '' -C "KubeCF CI #${{ github.run_id }}" -f ssh-key
+        ssh-add ssh-key
+        echo "::add-mask::$(cat ssh-key.pub)"
+        rm -f ssh-key ssh-key.pub
+        echo "::set-env name=SSH_AUTH_SOCK::${SSH_AUTH_SOCK}"
+        echo "::set-env name=SSH_AGENT_PID::${SSH_AGENT_PID}"
+
+    - name: set GKE_CRED_JSON
+      run: |
+        json_file="$(mktemp)"
+        echo "$GKE_CRED_JSON" > "${json_file}"
+        echo "::set-env name=GKE_CRED_JSON::${json_file}"
+      env:
+        GKE_CRED_JSON: ${{ secrets.GKE_CREDS_JSON }}
+
+    - name: cache.catapult-common-tools
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.workspace }}/catapult/modules/common/bin
+        key: ${{ runner.os }}-catapult-common-tools
+
+    - run: make k8s
+      working-directory: catapult
+      env:
+        GKE_CLUSTER_NAME: kubecf-ci-${{ github.run_id }}-${{ matrix.backend }}
+        OWNER: ${{ github.repository_owner }}
+        DEBUG_MODE: false
+        GKE_NODE_COUNT: 1
+        GKE_INSTANCE_TYPE: n1-highcpu-16
+
+    - name: export KUBECONFIG
+      run: |
+        set -o errexit -o nounset -o pipefail
+        cd "build${BACKEND}"
+        source .envrc
+        echo "::set-env name=KUBECONFIG::${KUBECONFIG}"
+        # Debugging aid; should be safe as the cluster will be torn down
+        # at the end of the run.
+        cat "${KUBECONFIG}" | gzip | base64 --wrap=0
+      working-directory: catapult
+
+    - name: Add tools to path
+      run: |
+        echo "::add-path::${TOOLS_DIR}"
+
+    - name: Generate KubeCF Configuration
+      run: |
```

Comment:
We shouldn't generate `values.yaml` anymore.
Do you want to make this change here (seems easier to me), or in a different PR?

Reply:
Let us handle it in different PR. I think it is more important to get initial commits on GHA which will enable us for quick iterations to improve it.

Reply:
Ok, can you please add an issue for it?

Reply:
https://github.com/cloudfoundry-incubator/kubecf/issues/1220